giskard/scanner/scanlogger.py

```diff
@@ -0,0 +1,65 @@
+import enum
+from typing import Optional
+
+from giskard.scanner.xprint import (
+    BLACK_STYLE,
+    BLUE_STYLE,
+    BOLD,
+    CHARS_LIMIT,
+    CYAN_STYLE,
+    GREEN_STYLE,
+    MAGENTA_STYLE,
+    RED_STYLE,
+    WHITE_STYLE,
+    YELLOW_STYLE,
+    Template,
+    xprint,
+)
+    
+class ScanLoggerLevel(enum.Enum):
+    DEBUG = 0
+    INFO = 5
+    CRITICAL = 10
+
+class ScanLogger:
+    level: ScanLoggerLevel = ScanLoggerLevel.INFO
+    
+    def setLevel(self, 
+        level: ScanLoggerLevel
+    ):
+        if not isinstance(level, ScanLoggerLevel):
+            raise ValueError("Invalid log level. Please provide a ScanLoggerLevel enum.")
+        self.level = level
+        
+    def checkLevel(self):
+        return self.level.name
```

Comment:
Is that needed? if you just used it to debug, let's remove it

Reply:
done!